frontend/src/context/config/sourceControl/sourceControlSlice.tsx

```diff
@@ -0,0 +1,31 @@
+import { createSlice } from '@reduxjs/toolkit'
+import { SOURCE_CONTROL_TYPES } from '@src/constants'
+
+export interface sourceControlState {
+  sourceControlConfig: { sourceControl: string; token: string }
+
+  isSourceControlVerified: boolean
+}
+
+export const initialSourceControlState: sourceControlState = {
+  sourceControlConfig: {
+    sourceControl: SOURCE_CONTROL_TYPES.GITHUB,
+    token: '',
+  },
+  isSourceControlVerified: false,
+}
+
+export const sourceControlSlice = createSlice({
+  name: 'sourceControl',
+  initialState: initialSourceControlState,
+  reducers: {
+    updateSourceControlVerifyState: (state, action) => {
```

Comment:
It's better to rename to updateVerifyState


And change others

Reply:
Good point!
but there are three different verifyStates, includes updateBoardVerifyState, updatePipelineToolVerifyState, and updateSourceControlVerifyState,  so maybe keep current name is ok /  make sense